opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/FamilyManager.java

```diff
@@ -670,6 +670,28 @@ private MyResourceId commonGetAnnotationSet(String id, @Nullable String studyStr
         return familyAclList;
     }
 
+    public List<QueryResult<FamilyAclEntry>> getAcls(String studyStr, List<String> familyList, boolean silent, String sessionId)
```

Comment:
In the rest of the managers, we would need to apply what I commented above for the CohortManager.